src/osw_ui.cpp

```diff
@@ -160,10 +163,10 @@ void OswUI::loop() {
         {
             std::lock_guard<std::mutex> notifyGuard(this->mNotificationsLock);
             // Draw all notifications
-            auto y = DISP_H - OswUINotification::sDrawHeight;
+            auto y = DISP_H;
```

Comment:
please rename `y`.

Reply:
What name would you propose? It _is_ literally the y coordinate being used here. Additionally, as this scope is only 5 lines long... What would be a better name?

Reply:
`y` It's simple, but I think it's best if it's a clear and instantly identifiable name that can be read without thinking in the distant future or when others see it.

Reply:
Using variable names like x and y is simple and intuitive, especially when they represent coordinates such as the x-axis and y-axis. However, if they appear in many places within the code, it can become harder to understand the overall context. 

Context Dependence: The meaning of x and y may only be clear from the surrounding context, making the code harder to read.
Potential Confusion: In more complex codebases, x and y might not always clearly convey their purpose.
Lack of Scalability: If additional coordinates or variables are introduced, temporary names like x1 or y1 may appear, reducing clarity.

But this case, It doesn't seem to have much effect on the current code, so looks good to me.